twitter4j-ads/src/twitter4jads/api/TwitterAdsPromotedTweetApi.java

```diff
@@ -29,7 +29,7 @@
      * @return Retrieve references to the Promoted Tweets associated with one or more line items.
      * @see <a href="https://dev.twitter.com/ads/reference/get/accounts/%3Aaccount_id/promoted_tweets">https://dev.twitter.com/ads/reference/get/accounts/%3Aaccount_id/promoted_tweets</a>
      */
-    BaseAdsListResponseIterable<PromotedTweets> getAllPromotedTweets(String accountId, boolean withDeleted,
```

Comment:
should we go ahead and use a builder pattern to create object Params? Something I was working till last week but haven't pushed changes yet. Optional makes the Arguments list longer and gets a little messy

Approving this